src/views/resources/form/orders.html

```diff
@@ -1239,6 +1240,59 @@ <h4 class="i18n">
           </footer>
         </div>
       </div>
+
+      <div class="card card-slided-up" id="subscription-invoices" style="display: none">
```

Comment:
```suggestion
      <div class="card card-slided-up" data-id="subscription-invoices" style="display: none">
```

Se usar ID fixo vai dar bug abrindo mais de um pedido ao mesmo tempo no painel